app/routes.py

```diff
@@ -1 +1,298 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, jsonify, request, make_response, abort
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from datetime import datetime 
+# from slack_sdk import WebClient
+import requests
+import os
+
+
+task_bp = Blueprint("task", __name__,url_prefix = "/tasks")
+
+# wave 1 
+# create a task
+@task_bp.route("", methods = ["POST"])
+def create_task():
+    response_body = request.get_json()
+    if "title" not in response_body or "description" not in response_body:
+        return jsonify({ "details": "Invalid data"}), 400
+
+    # using class method
+    new_task = Task.from_dict(response_body)
+
+    db.session.add(new_task)
+    db.session.commit() 
+
+    return jsonify({"task":{
+            "id": new_task.task_id,
+            "title": new_task.title,
+            "description": new_task.description,
+            "is_complete": False
+    }
```

Comment:
This would be a great place to use the `to_dict()` function you have in the Task model class.